src/Sylius/Bundle/OrderBundle/NumberGenerator/OrderNumberGeneratorInterface.php

```diff
@@ -11,13 +11,17 @@
 
 namespace Sylius\Bundle\OrderBundle\NumberGenerator;
 
+use Sylius\Component\Order\Model\OrderInterface;
+
 /**
  * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
  */
-interface SequentialOrderNumberGeneratorInterface
+interface OrderNumberGeneratorInterface
 {
     /**
+     * @param OrderInterface $order
+     *
      * @return string
      */
-    public function generate();
+    public function generate(OrderInterface $order);
```

Comment:
Why did you do this? It is just a generator and it does not care about an order itself.